README.md

```diff
@@ -1,3 +1,12 @@
+---
+description: prefect-aws provides pre-built Prefect tasks for working with Amazon Web Services.
+tags:
+    - overview
+    - getting started
+    - installation
+    - examples
+---
```

Comment:
This section gets rendered as a table by GitHub: 
![Screen Shot 2022-09-16 at 10 13 03 AM](https://user-images.githubusercontent.com/12350579/190672689-7a747bd7-0deb-4841-a26f-c40cdb7268c1.png)

I think the best course of action would be to inject this section into this page into the `gen_refs` script that allows us to use README.md as the index page for our docs.

Reply:
Ah I see my mistake here. We should remove this from the readme. My bad. It's fine on the other pages.